playground/backend/containers/java/build.gradle

```
@@ -71,7 +72,9 @@ docker {
   tags containerImageTags()
   buildArgs(['BEAM_VERSION': project.rootProject.hasProperty(["sdk-tag"]) ?
                            project.rootProject["sdk-tag"] :
-                           default_beam_version ])
+                           default_beam_version,
+             'GIT_COMMIT': getGitCommitHash(),
+             'GIT_TIMESTAMP': getGitCommitTimestamp()] )
```

Are there auto-formatters or at least guides? This looks random. Maybe at least add a trailing comma?
```suggestion
             'GIT_TIMESTAMP': getGitCommitTimestamp(),
  ])
```